tests/ApiTestCase.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests;
+
+use Coduo\PHPMatcher\Backtrace\VoidBacktrace;
+use Coduo\PHPMatcher\Factory\MatcherFactory;
+use Coduo\PHPMatcher\Matcher;
+use Coduo\PHPMatcher\PHPUnit\PHPMatcherAssertions;
+use PHPUnit\Framework\Attributes\Before;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+abstract class ApiTestCase extends WebTestCase
+{
+    use PHPMatcherAssertions;
+
+    protected KernelBrowser $client;
+
+    protected Matcher $matcher;
+
+    #[Before]
+    protected function setupClient(): void
+    {
+        $this->client = self::createClient();
+    }
+
+    #[Before]
+    protected static function _createMatcher(): Matcher
```

Comment:
If this change possible, the same as with `createClient/setupClient` change.

```suggestion
    protected static function createMatcher(): Matcher
    protected static function setupMatcher(): Matcher
```